htsget-http/README.md

```diff
@@ -38,7 +38,7 @@ These functions take query and endpoint information, and process it using [htsge
 This crate has the following features:
 * `s3-storage`: used to enable `S3Storage` functionality.
 * `url-storage`: used to enable `UrlStorage` functionality.
-* `experimental`: used to enable `C4GHStorage` functionality.
+* `experimental`: used to enable experimental features like `C4GHStorage`.
```

Comment:
experimental features (that are not defined in the official htsget/crypt4gh specification(s)?)... could be handy to clarify the level/type of "experimental"?